includes/adminpages.php

```diff
@@ -87,6 +87,7 @@ function pmpro_parent_file( $parent_file ) {
 		$parent_file = 'pmpro-dashboard';
 		$plugin_page = 'pmpro-dashboard';
 		$submenu_file = 'pmpro-membershiplevels';
+		$submenu_file .= '<span id="membershiplevels"></span>';
```

Comment:
Is this the only way to get an id into the menu? Can you select on something else. I'm surprised this doesn't break the menu when on a sub tab. Did you test that? Maybe we don't need to point exactly to this item if we can point to something close.

Reply:
Actually, we don't need this now, since we're not doing the step thru. I'll close this PR and put together a cleaner one.